packages/toolpad-app/src/toolpad/AppEditor/HierarchyExplorer/index.tsx

```diff
@@ -227,7 +227,14 @@ export default function HierarchyExplorer({ className }: HierarchyExplorerProps)
   );
 
   return (
-    <HierarchyExplorerRoot data-testid="hierarchy-explorer" className={className}>
+    <HierarchyExplorerRoot
+      sx={{
+        height: '100%',
+        overflowY: 'auto',
```

Comment:
Should this scroll horizontally as well?

Reply:
> Should this scroll horizontally as well?

So do you prefer `overflow: 'auto',`?

Reply:
this tree can nest arbitrarily deep, so there may be edge cases where the indentation surpasses the width of the sidebar. horizontal overflow makes for good UX in those cases I think.